databricks/koalas/plot.py

```diff
@@ -58,9 +58,7 @@ def get_top_n(self, data):
         max_rows = get_option("plotting.max_rows")
         # Simply use the first 1k elements and make it into a pandas dataframe
         # For categorical variables, it is likely called from df.x.value_counts().plot.xxx().
-        if isinstance(data, Series):
-            data = data.head(max_rows + 1).to_pandas().to_frame()
-        elif isinstance(data, DataFrame):
+        if isinstance(data, (Series, DataFrame)):
             data = data.head(max_rows + 1).to_pandas()
```

Comment:
BTW how come it is same? to_pandas previously always returned DataFrame but now seems it can return series too.

Reply:
ehh, i think the outcome of this `to_pandas` is either series or frame, both cases they can directly be used for plotting, so no need to convert to frame using `to_frame`. I tried a bit, and seems okay to me.